tools/jobify/jobify.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include "stljobs.h"
+#include <algorithm>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <Windows.h>
+
+wchar_t* get_subcommand() {

```

Comment:
I would put a comment clarifying what are we hunting for and an example string it works upon. 
Also, I would explain that it returns a pointer to a non-const `wchar_t` due to peculiarities of the CreateProcess API which insists on scribbling on its input arguments. :-)

Reply:
The non-const wchar_t are because that's what GetCommandLineW returns, not any particularity of CreateProcess (although working with CreateProcess is a nice side benefit of that)